packages/probers/cypress/e2e/signUp.cy.ts

```diff
@@ -102,10 +102,36 @@ describe('Sign Up', () => {
         }
       )
 
-      cy.findByRole('textbox', { name: /display name/i }).type(handle)
+      cy.findByRole('textbox', { name: /display name/i }).type(name)
       cy.findByRole('button', { name: /continue/i }).click()
 
       cy.findByRole('heading', { name: /select your genres/i }).should('exist')
+      cy.findByText(/start by picking some of your favorite genres./i).should(
+        'exist'
+      )
+
+      cy.findByText(name).should('exist')
+      cy.findByText(handle).should('exist')
+
+      const genres = [/^acoustic/i, /^pop/i, /^lo-fi/i, /^electronic/i]
+
+      for (const genre of genres) {
+        cy.findByRole('checkbox', { name: genre }).check()
+      }
+
+      cy.findByRole('button', { name: /continue/i }).click()
+
+      cy.findByRole('heading', {
+        name: /follow at least 3 artists/i,
+        level: 1
+      }).should('exist')
+      cy.findByText(/curate your feed with tracks uploaded/i).should('exist')
```

Comment:
wondering if we should get in the habit of exporting our messages object and referencing that here?
It would make the tests less readable but also less brittle

Reply:
That is totally reasonable, would be interested in that. I think in a way it's a catch to make sure the text changes are intentional, but yeah very open to this discussion